### ISSUE TITLE
Scala 2.13 project navigates to Scala 2.12 sources

### ISSUE THREAD

**Issue by mario-galic** [scalameta/metals#2738]
**Describe the bug**

Freshly imported Scala 2.13 project navigates to Scala 2.12 sources after control-clicking (or hovering) the symbol. 

Possible relevant log messages:

```
2021.04.24 14:29:41 WARN  Build server is not auto-connectable.
2021.04.24 14:29:44 INFO  running '/usr/lib/jvm/java-8-openjdk-amd64/bin/java -Djline.terminal=jline.UnsupportedTerminal -Dsbt.log.noformat=true -Dfile.encoding=UTF-8 -jar /tmp/metals4954606553543472088/sbt-launch.jar -Dbloop.export-jar-classifiers=sources bloopInstall'
2021.04.24 14:29:45 WARN  no build target for: /home/******/sandbox/scala2-hello-world/project/metals.sbt
2021.04.24 14:29:45 WARN  no build target for: /home/******/sandbox/scala2-hello-world/project/project/metals.sbt
2021.04.24 14:29:45 INFO  skipping build import with status 'Started'
2021.04.24 14:29:47 INFO  [info] welcome to sbt 1.5.0 (Private Build Java 1.8.0_282)
2021.04.24 14:29:47 INFO  no build target: using presentation compiler with only scala-library: 2.12.13
```

1. `sbt new scala/scala-seed.g8`
2. `code .`
3. Control+click on `map` symbol of `List(42).map(identity)`
4. It navigates to [2.12](https://github.com/scala/scala/blob/2.12.x/src/library/scala/collection/immutable/List.scala#L290) source code of `map`  instead of [2.13](https://github.com/scala/scala/blob/d974e9960081eace9c54c7fddbd15d91ce4847a8/src/library/scala/collection/immutable/List.scala#L244)

**Expected behavior**

It should navigate to source code of Scala version specified in `build.sbt`

**Screenshots**

![image](https://user-images.githubusercontent.com/13835317/115960970-31492580-a50c-11eb-801c-d7910ed56726.png)

![image](https://user-images.githubusercontent.com/13835317/115961239-7b7ed680-a50d-11eb-90fa-2ca39b0550b4.png)

**Installation:**

- Operating system: Linux
- Editor: Visual Studio Code
- Metals version: v0.10.2

**Additional context**

Compiling, running and testing works fine from VSC.

**Search terms**

Goto definition

**Comment by dos65** [comment]
Thanks for reporting!

Actually, there is a fix for that problem in #2668 and it will be available in the next release.

**Comment by tgodzik** [comment]
Fixed in #2668
